tests/tool/Integration.Tests/E2ETest.cs

```diff
@@ -26,10 +26,12 @@ public class E2ETest
             ".upgrade-assistant"
         };
 
-        [InlineData("AspNetMvcTemplate", "TemplateMvc.csproj", "")]
-        [InlineData("WpfSample", "BeanTrader.sln", "BeanTraderClient.csproj")]
+        [InlineData("AspNetMvcTemplate.", "csharp", "TemplateMvc.csproj", "")]
```

Comment:
Stray '.' atthe end of the AspNetMvcTemplate scenario name

Reply:
Also, maybe we should rename that scenario AspNet since it's no longer just MVC templates.